storage/tests/unit/test_client.py

```diff
@@ -409,7 +409,42 @@ def test_create_bucket_conflict(self):
         json_sent = http.request.call_args_list[0][1]["data"]
         self.assertEqual(sent, json.loads(json_sent))
 
-    def test_create_bucket_success(self):
+    def test_create_bucket_with_object_conflict(self):
+        from google.cloud.exceptions import Conflict
+        from google.cloud.storage.bucket import Bucket
+
+        PROJECT = "PROJECT"
```

Comment:
nit: since these are just local variables, use lower-case letters.

Reply:
Changed variable names to lower-case on all tests I'm working on.